cypress/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM cypress/included:5.3.0
+FROM cypress/included:6.6.0
```

Comment:
I know @jekabs-karklins mentioned that there were some issues with this 6.6.0 cypress version on the user-office-frontned but maybe that is not the case here.

Reply:
Yes, I fixed it, the cypress package.json had some missing dependencies